CBLClient/Database.py

```diff
@@ -230,15 +230,20 @@ def create_bulk_docs(self, number, id_prefix, db, channels=None, generator=None,
                 doc_body = doc_generators.simple_user()
             elif generator == "complex_doc":
                 doc_body = doc_generators.complex_doc()
+            elif generator == "20mb":
+                doc_body = doc_generators.doc_20mb()
             else:
                 doc_body = doc_generators.simple()
 
             if channels is not None:
                 doc_body["channels"] = channels
 
             if attachments_generator:
-                types.verify_is_callable(attachments_generator)
-                attachments = attachments_generator()
+                if attachment_file_list is not None:
+                    attachments = attachments_generator(attachment_file_list)
```

Comment:
what type is attachments_generator? I kind of have problem to imagine what it would be during runtime. I don't see any example in existing code for passing parameters to attachments_generator, all I got were attachments_generator()

Reply:
attachment generator are functions defined to generate different type of attachment defined in attachment.py. One of function (load_from_data_dir) is to load attachment from a given path for which we didn't provide a provision for in create_bulk docs, so I've added additional argument, attachment_file_list.